recipe/meta.yaml

```diff
@@ -23,7 +23,11 @@ source:
   sha256: 8fe2d344463dbefc2db4239a4203a55ed0324faceaae57276a40c4fabda84c37  # [win64]
 
 build:
-  number: 2
+  number: 3
+  run_exports:
+    # seems to be minor version compatibility
+    # https://abi-laboratory.pro/tracker/timeline/ffmpeg/
+    - {{ pin_subpackage('ffmpeg', max_pin='x.x') }}
```

Comment:
Please don't add the run_exports in this version. Since other build numbers did not have it, it's not easy to enforce which version of ffmpeg to use in other recipes (e.g. opencv) to make sure a package with run_exports is being used.

Reply:
ok, removing it